packages/e2e/cypress/e2e/api/sqlRunner.cy.ts

```diff
@@ -92,6 +93,97 @@ Object.entries(warehouseConnections).forEach(
                     expect(fieldResults.order_date).to.be.eq('date');
                 });
             });
+
+            it(`Get streaming results from ${warehouseName}`, () => {
+                const [database, schema] = getDatabaseDetails();
+                const selectFields =
+                    warehouseConfig.type !== WarehouseTypes.SNOWFLAKE
+                        ? `*`
+                        : `payment_id as "payment_id", 
+                amount as "amount", 
+                payment_method as "payment_method"`; // Need to lowercase column ids in snowflake
+                const sql = `SELECT ${selectFields}
+                     FROM ${database}.${schema}.payments ORDER BY payment_id asc LIMIT 2`;
+
+                cy.request({
+                    url: `${apiUrl}/projects/${projectUuid}/sqlRunner/run`,
+                    headers: { 'Content-type': 'application/json' },
+                    method: 'POST',
+                    body: JSON.stringify({
+                        sql,
+                    }),
+                }).then((runResp) => {
+                    expect(runResp.status).to.eq(200);
+                    const { jobId } = runResp.body.results;
+
+                    const maxRetries = 20; // Snowflake is a bit slow compared to the others (normally takes ~3 seconds)
+
+                    // Poll request until job is `completed`
+                    const poll = (retries = 0) => {
```

Comment:
this is a method to wrap the `job` status request, and call it recursively based on the `status` . 

Postgres (locally) takes <100ms to complete jobs, while snowflake takes ~3sec. I don't want to force cy.wait(5s) just to make it pass.